GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -270,5 +272,50 @@ private void KeyPressed(object sender, KeyEventArgs e)
                 CancelButton_Click(sender, e);
             }
         }
+
+        internal SearchModifications SearchMod = new SearchModifications();
+
+        private void TextChanged_Fixed(object sender, TextChangedEventArgs args)
+        {
+            SearchMod.FixedSearch = true;
+            SetTimer();
```

Comment:
remove all this timer stuff, I assume that's for debug purposes?

Reply:
I’m using the timer to keep track of the user typing. Without it, the tree gets filtered immediately after each keystroke and makes the search slow. Should I make a “go” button instead?

Reply:
that's fine then - can you put a comment in there explaining what that's for?

Reply:
yeap. I’ll add comments and make the other edits first thing tomorrow!